src/projects/actions.js

```diff
@@ -12,6 +12,65 @@ import {
 } from './action-types';
 import { SELECT_PROJECT } from "./action-types";
 import {firebaseDb} from "../firebase";
+import { getCookie, getUrlSearchParams } from "../utils/browser-utils";
+import { getAuth } from "../auth";
+import history from "../history";
+
+
+export const initProject = () => (dispatch, getState) => {
```

Comment:
Where do you push the user to the projects screen in case the project not exists?

Reply:
We do it in the tasks-page.js in case of a bad project
I didn't want to use this function to do it, it can bo too generic and there might be some use cases where you don't want to redirect